docs/application/native/guides/app-management/app-controls.md

```diff
@@ -664,6 +668,120 @@ create_base_gui(appdata_s *ad)
    }
    ```
 
+<a name="action"></a>
+## Handlng Application Control Actions
```

Comment:
Please fix the type. 
Handling

Reply:
> Please fix the type.
> Handling

@hjhun Kindly fix the typo as mentioned in the comment by @mijin85cho.  Please write "Handling" in place of "Handlng"

Reply:
I've already fixed it.

Reply:
@hjhun An 'i' is missing before 'ng' so currently the spelling is 'Handlng' whereas it should be 'Handling'. Please check for the same.

Reply:
oops. It's my mistake. I've updated it locally.